analysis/tests/src/expected/SignatureHelp.res.txt

```diff
@@ -10,12 +10,11 @@ ContextPath Value[someFunc]
 Path someFunc
```

Comment:
Ignore this file, we checked it in the last PR. I just removed the dedicated uncurried test file here.